src/NHSD.BuyingCatalogue.Organisations.Api/appsettings.json

```diff
@@ -10,8 +10,26 @@
     }
   },
   "AllowedHosts": "*",
-
   "ConnectionStrings": {
     "CatalogueUsers": "Data Source=localhost,1533;Initial Catalog=CatalogueUsers;MultipleActiveResultSets=true;User ID=NHSD;Password=DisruptTheMarket1!"
+  },
+  "SmtpServer": {
+    "Host": "localhost",
+    "Port": 1025,
+    "UseSsl": false,
+    "Authentication": {
+      "IsRequired": false
+    }
+  },
+  "Registration": {
+    "EmailMessage": {
+      "Sender": {
+        "Name": "Buying Catalogue Team",
+        "Address": "noreply@buyingcatalogue.nhs.uk"
+      },
+      "Subject": "Set password for Buying Catalogue",
+      "HtmlBody": "<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Strict//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\"><html xmlns=\"http://www.w3.org/1999/xhtml\"><head><meta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\" /><title></title></head><body><p>Thank you for requesting a user account for the Buying Catalogue.</p><p>Your account has now been created and you can use <a href=\"[ResetPasswordLink]\">this link</a> to set your password.</p><p>Your password must be at least ten characters in length and match on confirmation.</p><p>It must contain at least 3 of the following 4 character types:</p><p>lower case letters (a-z)<br />upper case letters (A-Z)<br />numbers (0-9)<br />special characters (e.g. !@#$%^&amp;*)</p><p>The link can be used only once to set up one account, and will be valid for 24 hours from the time you received this email.</p><p>Yours sincerely,<br />The Buying Catalogue team</p></body></html>",
+      "TextBody": "Thank you for requesting a user account for the Buying Catalogue.\n\nYour account has now been created and you can use this link to set your password.\n\n[ResetPasswordLink]\n\nYour password must be at least ten characters in length and match on confirmation.\n\nIt must contain at least 3 of the following 4 character types:\n\nlower case letters (a-z)\nupper case letters (A-Z)\nnumbers (0-9)\nspecial characters (e.g. !@#$%^&*)\n\nThe link can be used only once to set up one account, and will be valid for 24 hours from the time you received this email.\n\nYours sincerely,\nThe Buying Catalogue team"
```

Comment:
I wonder whether this is the optimal way to set the content of the email. Would it perhaps be better to obtain the content through a reading a file?

Reply:
I did ponder this implementation a little. My very first implementation just relied on constants but I thought it would be better to allow the message content to be configurable given how often the content has change on the FE.

While app settings isn't really intended for content management our content is relatively simple and we get well-understood configuration management out of the box. Given that the default content is provided via `appSettings.json` I'm not sure placing the content in a different file would bring any performance benefit – particularly as the content is currently only read once on startup.

However, I recognize there is room for improvement here so if you or anyone else has any suggestions please let me know.

Reply:
This is fine for now, isn't that pleasant to look at or I think that easy to manage in this form. I'd probably want a HTML file so I can design / style the email template. We do have a database to store things, again not that I am suggesting we use that, but wouldn't have to worry about backing up the data. It kind of depends what sort of functionality the client wants going forward e.g. different email templates per user type? / dynamic content. Simplest approach for now, enhance functionality at a later date.